public/video-ui/src/services/pandaReqwest.js

```diff
@@ -42,6 +42,15 @@ export function pandaReqwest(reqwestBody, timeout = 0) {
 
     // prettier-ignore
     if (payload.contentType === 'application/json' && typeof payload.data === 'object') {
+
+      //Replace all empty strings with nulls so that dynamo can save all the data we
+      //send to the server
+      Object.keys(payload.data).forEach(key => {
```

Comment:
Could we do this at the component or placeholder level? Seems easy to break in the future if it's done here